view/src/components/layout/MainLayout/MainLayout.tsx

```diff
@@ -1,11 +1,13 @@
 import React, { FC } from 'react'
 import Header from '@components/common/Header/AdminHeader'
 import Footer from '@components/common/Footer/Footer'
+import EventParticpantCard from '@components/common/EventParticpantCard'
 
 const MainLayout: FC = () => {
   return (
     <div>
       <Header />
+      <EventParticpantCard  />
```

Comment:
event idをpropsで受け取る場合以下のようになります！！

```suggestion
      <EventParticipantCard eventId='1' />
```

↑これも実際は`1`じゃなく、そのページからidを取得して変数を受け取る形になりますが、今はとりあえず1を入れています。